test/controllers/users_controller_test.js

```diff
@@ -254,15 +311,15 @@ describe("Endpoints exists for users", () => {
             chai.request(usersRoutes).
                 post("/users/likes_recipe").
                 set("content-type", "application/json").
-                set("token", userToken).
+                set("token", userToken.token).
```

Comment:
because usertoken is an object with token as value and we want to use the value instead

Reply:
If userToken is an object, then I think we should do some object destructuring to make our code a bit cleaner. Maybe around line 311, add `let { token } = userToken;` and replace all instances of `userToken.token`